sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/AsyncVoidMethod.cs

```diff
@@ -1,18 +1,31 @@
 ﻿using System;
+using System.Threading.Tasks;
 
 namespace Tests.Diagnostics
 {
     public class Foo : EventArgs { }
 
     public class EventHandlerCases
     {
-        async void MyMethod() { } // Noncompliant {{Return 'Task' instead.}}
+        async void MyMethod() {  // Noncompliant {{Return 'Task' instead.}}
 //            ^^^^
-        async void MyMethod(object sender, EventArgs args) { }
+            await Task.Run(() => Console.WriteLine("test"));
+        }
+
+        async void MyMethod(object sender, EventArgs args)
+        {
+            await Task.Run(() => Console.WriteLine("test"));
+        }
 
-        async void MyMethod1(object o, EventArgs e) { }
+        async void MyMethod1(object o, EventArgs e)
```

Comment:
Isn't `MyMethod1` almost the same with `MyMethod` and just parameter names changed? Can we remove it?